### DISCUSSION TITLE
1.4.2 코드진행중 에러

### DISCUSSION THREAD

**Discussion by leech1211** [velopert/dealing-with-react-native#16] [Q&A] [answered]
1.4.2 부분까지 진행하던중 에러가 생겼습니다
index.js와 App.js 의 내용은 책과 동일합니다 터미널에 yarn android 를 이용해 실행을 시키면 \

![image](https://user-images.githubusercontent.com/55794835/149864705-e9d784b9-3f28-42ac-97ce-b49ecd51685e.png)

다음과 같은 오류가 발생합니다 구글링을 해보니 경로를 다시 잡아주어야 한다고 하는데 
어떻게 해결해야할까요?ㅜ

**Comment by velopert** [comment, marked as answer]
늦게 답변하게 되어 죄송합니다.
나중에 이 이슈를 누군가 또 겪게 되시면,

Node.js 를 v18으로 올려주시고,
프로젝틀르 새로 생성해보세요.
